app/graphql/albums/queries.js

```diff
@@ -1,20 +1,20 @@
 const { gql } = require('apollo-server'),
-  { getAlbum, getAlbumPhotos } = require('../../services/album');
+  { getAlbum, getAlbums } = require('./resolver'),
+  { DEFAULT_OFFSET, DEFAULT_LIMIT, DEFAULT_SORTKEY } = require('../../constants');
 
 module.exports = {
   queries: {
-    album: async (_, params) => {
-      const album = await getAlbum(params);
-      const albumPhotos = await getAlbumPhotos(params.id);
-      album.artist = album.userId;
-      delete album.userId;
-      album.photos = albumPhotos;
-      return album;
-    }
+    album: (_, params) => getAlbum(_, params),
+    albums: (_, params) => getAlbums(_, params)
   },
   schema: gql`
```

Comment:
Ok, now that you get it, we will rice the complexity a little. 

- First, search about Inputs in GraphQL.
- Create an Input for filtering and another for sorting.
- SortingInput should have the field and the order.
- FilterInput should have the field and the value to search.
- Please implement them in the `albums` query.

Tell me if you have ani doubt